taggit/models.py

```diff
@@ -68,8 +68,11 @@ def save(self, *args, **kwargs):
             except IntegrityError:
                 pass
             # Now try to find existing slugs with similar names
-            slugs = set(Tag.objects.filter(slug__startswith=self.slug)
-                                   .values_list('slug', flat=True))
+            slugs = set(
+                self._default_manager
```

Comment:
Can you link to one of those? I'm not as familiar with django internals, and most of the ones I see with a simple search show `model._default_manager`.